src/docs/doxygen/GeosxConfig.hpp

```diff
@@ -141,7 +141,7 @@
 #define fmt_VERSION 8.0.1
 
 /// Version information for python
-/* #undef Python3_VERSION */
+#define Python3_VERSION 3.8.5
```

Comment:
I'm surprised that this file is committed...
Where do you get this `3.8.5` from? Is it generated through `GeosxConfig.in`?

Reply:
Yeah, I have seen this happen before on occasion...  It is definitely generated as part of the config/build process, and we should try to squash it.

Reply:
@TotoGaz - I've chatted with @rrsettgast about this, and he is planning to look into it